atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -0,0 +1,241 @@
+        Note:
+            - Modifies both TEXT and CLUSTERCENTER collections
+            - Preserves the text's previous competency associations
+            - Updates cluster medoid using a weighted average approach
+            - All changes are atomic - either all succeed or none are applied
+
+        Warning:
+            Ensure both text_id and competency_id exist in the database before calling
+            this function to avoid potential errors.
+        """
+
+        text = self.weaviate_client.get_embeddings_by_property(CollectionNames.TEXT.value, "text_id", text_id)
```

Ruff (0.11.9)

Line too long (110 > 88)

(E501)